src/redux/reducers/skills.js

```diff
@@ -27,6 +27,7 @@ const defaultState = {
   ratingRefine: null,
   reviewed: false,
   staffPicks: false,
+  reviewedClicked: false,
```

Comment:
No needed

Reply:
@fragm3 I wanted to check `show only reviewed skills` only if the number of reviewed skills is more than 1. In our current implementation when we check on `show only reviewed skills` manually first an action is called to set reviewed state in reducers and then the cards are loaded. So I want to check `show only reviewed skills` only initially and then user can have freedom to check, uncheck. So for this I thought of many logics but came up with this of using a reviewedClick state. So basically reviewedClick state is initially set to false and once a user clicks on `show only reviewed skills` I set this to true so that while loading cards I can avoid the `show only reviewed skills` from being checked.

I'll look into the console error

Reply:
@harsh-jindal it essentially is a filter, we have `reviewed` in store, you can use it to achieve the same.